code/game/gamemodes/colonialmarines/colonialmarines.dm

```diff
@@ -285,7 +286,10 @@
 			if(SSticker.mode && SSticker.mode.is_in_endgame)
 				round_finished = MODE_INFESTATION_X_MINOR //Evacuation successfully took place.
 			else
+				SSticker.roundend_check_paused = TRUE
 				round_finished = MODE_INFESTATION_M_MAJOR //Humans destroyed the xenomorphs.
+				addtimer(CALLBACK(SSticker, TYPE_PROC_REF(/datum/controller/subsystem/ticker, toggle_roundend_check)), MARINE_MAJOR_ROUND_END_DELAY)
```

Comment:
Alternatively, use the varset callback instead of the proc call

Reply:
Yeah just gonna use varset. This variable isn't used anywhere else anyhow.